migrations/1677284495299_locations.ts

```diff
@@ -6,15 +6,18 @@ export const shorthands: ColumnDefinitions | undefined = undefined;
 export function up(pgm: MigrationBuilder): void {
   pgm.createTable('locations', {
     id: {
-      type: 'serial',
+      type: 'bigserial',
       primaryKey: true,
     },
     inscription_id: {
-      type: 'int',
+      type: 'bigint',
```

Comment:
an `inscription_id` is not a `int`/`bigint`, it's a `string`, no?

Reply:
Ah, yep, in the API `inscription_id` is the id on the postgres table `inscriptions`... We call the other one `genesis_id`